.github/workflows/build.yaml

```diff
@@ -35,6 +35,41 @@ jobs:
         with:
           node-version: "16"
 
+      - name: Fetch dependencies from cache
+        id: cache-node-modules
+        uses: actions/cache@v3
+        with:
+          path: "node_modules"
```

Comment:
✍🏼 Since these jobs like `fmt` don't require any dependencies besides the root ones, I left the caching to only these `node_modules`. Happy to change back if desired.